src/main/resources/logback.xml

```diff
@@ -7,17 +7,8 @@
 
 <configuration>
     <timestamp key="bySecond" datePattern="yyyyMMdd'T'HHmmss"/>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <!-- encoders are assigned the type
-             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
-        <encoder>
-            <pattern>%highlight%d{HH:mm:ss.SSS} %-5level %logger{36} - %msg%n</pattern>
-        </encoder>
-        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-            <level>INFO</level>
-        </filter>
-    </appender>
     <!-- File appender is configured and added by edu.ie3.simona.logging.logback.LogbackConfiguration -->
+    <!-- Console appender is configured and added by edu.ie3.simona.logging.logback.LogbackConfiguration -->
     <root level="debug">
         <appender-ref ref="STDOUT"/>
```

Comment:
This appender now does not exist anymore, right?

Also, do we need this file anymore? There's not many lines left in here

Reply:
If we remove the file, the configuration of the config log level is no longer working.